lib/component.js

```diff
@@ -361,6 +395,17 @@ class Component extends WebComponent {
     });
   }
 
+  /**
+   * Enqueue a function to be run at the end of the components disconnectedCallback.
+   */
+  onDisconnected(fn) {
```

Comment:
move `onDisconnected` and `onConnected` alphabetically in the custom public methods section, ~L150 between `navigate` and `setConfig`